### ISSUE TITLE
M26 S7203914 figures wrong offset.

### ISSUE THREAD

**Issue by gddeen** [MarlinFirmware/Marlin#3711]
M27 shows it didn't see a valid value. ...

**Comment by gddeen** [comment]
There are two places that use card.setIndex(code_value_short()).
M24 / M26

**Comment by Blue-Marlin** [comment]
`code_value_short()` indeed seems to be a bit short for a byte index into a file.

```
  FORCE_INLINE void setIndex(long index) { sdpos = index; file.seekSet(index); }
```

long seems to be the better choice.
Try `code_value_long()` instead.

**Comment by thinkyhead** [comment]
Patched up with #3713!
